config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,372 @@
+# frozen_string_literal: true
+
+# These defaults are defined and maintained by the community at
+# https://github.com/heartcombo/simple_form-bootstrap
+# Please submit feedback, changes and tests only there.
+
+# Uncomment this and change the path if necessary to include your own
+# components.
+# See https://github.com/heartcombo/simple_form#custom-components
+# to know more about custom components.
+# Dir[Rails.root.join('lib/components/**/*.rb')].each { |f| require f }
+
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
You have two `SimpleForm.setup` calls in two different files. Are you sure they are not conflicting with each other?

Reply:
Yes, they are not conflicting, because they are both was created by generating from gem

Reply:
:thinking: OK, need to re-check that. I'll ping you later.